x-pack/plugins/security_solution/public/management/pages/event_filters/service/index.ts

```diff
@@ -90,6 +90,19 @@ export class EventFiltersHttpService implements EventFiltersService {
   async updateOne(
     exception: Immutable<UpdateExceptionListItemSchema>
   ): Promise<ExceptionListItemSchema> {
+    // Clean unnecessary fields for update action
+    [
+      'created_at',
+      'created_by',
+      'created_at',
+      'created_by',
+      'list_id',
+      'tie_breaker_id',
+      'updated_at',
+      'updated_by',
+    ].forEach((field) => {
+      delete exception[field as keyof UpdateExceptionListItemSchema];
```

Comment:
Can we change the implementation so that we don't mutate the object that was passed on input to this method. I think that maybe this might work: 

```typescript
async updateOne(
   {
      created_at,
      created_by,
      created_at,
      creatd_by,
      list_id,
      tie_breaker_id,
      updated_at,
      updated_by,
      ...exception
    }: Immutable<UpdateExceptionListItemSchema>
  ): Promise<ExceptionListItemSchema> {
```